lib/Analysis/DxilValueCache.cpp

```diff
@@ -232,6 +233,20 @@ Value *DxilValueCache::SimplifyAndCacheResult(Instruction *I, DominatorTree *DT)
       llvm::Type *i1Ty = llvm::Type::getInt1Ty(M->getContext());
       Simplified = llvm::ConstantInt::get(i1Ty, 1);
     }
+    else {
+      SmallVector<Value *,16> Args;
+      for (unsigned i = 0; i < CI->getNumArgOperands(); i++) {
+        Args.push_back(TryGetCachedValue(CI->getArgOperand(i)));
+      }
+
+      Function *Callee = nullptr;
```

Comment:
Minor, but it seems since getCalledFunction() is called either way, it might as well be used to initialize Callee here

Reply:
Good point. Changed.